Source/Core/Core/Boot/Boot_BS2Emu.cpp

```diff
@@ -328,25 +326,25 @@ bool CBoot::SetupWiiMemory(u64 ios_title_id)
 // Wii Bootstrap 2 HLE:
 // copy the apploader to 0x81200000
 // execute the apploader
-bool CBoot::EmulatedBS2_Wii()
+bool CBoot::EmulatedBS2_Wii(const DiscIO::IVolume* volume)
```

Comment:
Should this take a ref instead to indicate that EmulatedBS2_Wii should be called with a valid volume, since BS2 technically only runs for discs?